lib/index.js

```diff
@@ -185,10 +186,28 @@ function stringifyTopojson(layers){
 
 Tilesplash.prototype.layer = function(name){
   var callbacks = Array.prototype.slice.call(arguments);
+
+  // shave off name
   callbacks.shift();
+
+  var mvtOptions = {};
+  // if options were passed, we'll have 2 left
+  if (callbacks.length === 2){
```

Comment:
Just trying to get access to the passed in mvtOptions. Using `argumants` makes this tricky, and I'm unsure on the best way to do it.